source/incident-collaboration/overview.rst

```diff
@@ -1,23 +1,22 @@
 Overview
 ========
 
-Mattermost Incident Collaboration equips your team to coordinate, manage, and resolve incidents from within Mattermost using playbooks. Each playbook represents a recurring outcome that your teams collaborate to achieve, such as Service Outage Recovery or Customer Onboarding. For the participants, it prescribes processes such as task checklists, status updates, and retrospective reports. For the integrated tools, it configures the triggers to perform automate actions. And for stakeholders, it provides a single pane of glass for visibility into each run as well as aggregate insights over time.
+Mattermost Playbooks are a collaboration tool for teams. Teams run a playbook every time they want to orchestrate people, tools, and data to achieve that outcome as quickly as possible while providing visibility to stakeholders. Each playbook represents a recurring outcome or specific goal that your teams collaborate on to achieve, such as service outage recovery or customer onboarding. For the participants, it prescribes processes such as task checklists, status updates, and retrospective reports. For the integrated tools, it configures the triggers to perform automated actions. And for stakeholders, it provides a single pane of glass for visibility into each run as well as aggregate insights over time.
```

Comment:
I still see "that outcome" before defining what the outcome is, do we want to reorder it?

Reply:
Oops! Thanks - I left this out by mistake!